src/store/global-provider.tsx

```diff
@@ -1,22 +1,61 @@
 import React from 'react'
+import { GrowthBook, GrowthBookProvider } from '@growthbook/growthbook-react'
 import { BreakpointsProvider } from './breakpoints-context'
 import { PopupProvider } from './popup-context'
 import { RegionProvider } from './region-context'
 import { WebsiteStatusProvider } from './website-status-context'
 import { MediaContextProvider } from 'themes/media'
+import { getClientInformation, getDomain, getLanguage } from 'common/utility'
+import { growthbook_client_key } from 'common/constants'
 
 type GlobalProviderProps = { children: React.ReactNode }
 
+let growthbook
+if (typeof window !== 'undefined') {
+    import('@deriv/analytics').then(({ RudderStack }) => {
+        const anonymousId = RudderStack.getAnonymousId()
+        growthbook = new GrowthBook({
+            apiHost: 'https://cdn.growthbook.io',
+            clientKey: growthbook_client_key,
+            enableDevMode: true,
+            attributes: {
+                id: anonymousId,
+            },
+            trackingCallback: (experiment, result) => {
+                RudderStack.track('Experiment Viewed', {
+                    experimentId: experiment.key,
+                    variationId: result.variationId,
+                })
+            },
+        })
+        growthbook.loadFeatures({
+            autoRefresh: true,
+        })
+
+        const language = getLanguage()
+        const domain = getDomain()
+        const client_information = getClientInformation(domain)
+
+        if (client_information) {
+            RudderStack.identifyEvent(client_information.loginid, {
+                language,
+            })
+        }
+    })
+}
```

Comment:
@markwylde-deriv Would you please move these into a separate provider file 🙏🏻 I believe `GlobalProvider` is just supposed to wrap the whole app with all the global providers and not contain anything itself 🤔

Reply:
Thanks @farzin-deriv. Absolutely. Fixed in e40166cecbe57dab54be2164ea9b2dfc18d319e0.